commit 00dc519c52a2eb4ae595cbd3fa3b50b48d11a1c2
Author: Devesh Sharma <devesh.sharma@broadcom.com>
Date:   Fri Mar 22 11:06:59 2019 -0400

    add gen p5 chip pci id to ini file
    
    Adding the PF pci id for new gen p5 chip to openmpi code.
    There is no PR associate to master in this patch because
    in master openib BTL is deprecated. This patch should be
    cherry picked to 4.0.x as well.
    
    Signed-off-by: Devesh Sharma <devesh.sharma@broadcom.com>

diff --git opal/mca/btl/openib/mca-btl-openib-device-params.ini opal/mca/btl/openib/mca-btl-openib-device-params.ini
index 4a0a62467d46..51b2470b757b 100644
--- opal/mca/btl/openib/mca-btl-openib-device-params.ini
+++ opal/mca/btl/openib/mca-btl-openib-device-params.ini
@@ -336,7 +336,7 @@ max_inline_data = 72
 
 [Broadcom BCM57XXX]
 vendor_id = 0x14e4
-vendor_part_id = 0x1605,0x1606,0x1614,0x16c0,0x16c1,0x16ce,0x16cf,0x16d6,0x16d7,0x16d8,0x16d9,0x16df,0x16e2,0x16e3,0x16e5,0x16eb,0x16ed,0x16ef,0x16f0,0x16f1
+vendor_part_id = 0x1605,0x1606,0x1614,0x16c0,0x16c1,0x16ce,0x16cf,0x16d6,0x16d7,0x16d8,0x16d9,0x16df,0x16e2,0x16e3,0x16e5,0x16eb,0x16ed,0x16ef,0x16f0,0x16f1,0x1750
 use_eager_rdma = 1
 mtu = 1024
 receive_queues = P,65536,256,192,128
